src/event_pusher/mod_event_pusher_push.erl

```diff
@@ -105,8 +105,9 @@ push_event(_, _) ->
     ok.
 
 do_push_event(Host, #chat_event{from = From, to = To, packet = Packet}) ->
-    mod_event_pusher_push_plugin:should_publish(Host, From, To, Packet) andalso
-        publish_message(From, To, Packet).
+    mod_event_pusher_push_plugin:should_publish(Host, From, To, Packet)
```

Comment:
can we add a comment?
- do not push messages without body element
- but push messages with empty body element